CONTRIBUTING.md

```diff
@@ -5,11 +5,61 @@ We love to pull requests from everyone! We follow the standard Git workflow of `
+
+### 1. Join the Community & Learn How Things Work
+- **Join Slack**: Visit [https://chaoss.community/kb-getting-started/](https://chaoss.community/kb-getting-started/) and join the `#wg-augur-8knot` channel
+- **Read CHAOSS Getting Started Docs**: Explore the Code of Conduct, practitioner guides, and other interesting materials on the CHAOSS website
+- **Review this CONTRIBUTING.md**: Make sure you understand our contribution guidelines, including our AI contribution policy
+- **Attend Meetings**: Find the calendar and join some meetings to learn about the community. We recommend:
+  - Newcomer Hangout
+  - Community Call
+  - Software Working Group
+  - Data Science Working Group
+- **Watch Meeting Recordings**: Check out the back-catalog of meeting recordings on the [CHAOSS YouTube channel](https://www.youtube.com/@CHAOSStube)
+
+### 2. Run Augur
+- **Explore the Documentation**: Peruse the docs at [https://oss-augur.readthedocs.io/en/main/](https://oss-augur.readthedocs.io/en/main/), especially the Docker setup guide
+- **Get Augur Running**: Follow the installation instructions to get Augur running on your machine (there will soon be a YouTube video on the CHAOSS channel showing how to set up Augur in about 10 minutes)
+- **Report Issues with Documentation**: If you get stuck because the docs are confusing or outdated, let someone know on Slack, open an issue, or raise a PR to fix it!
+
+### 3. Start with Good First Issues
+- **Browse Good First Issues**: Look through issues labeled [`good first issue`](https://github.com/chaoss/augur/labels/good%20first%20issue) in the Augur repo
+- **Try First Timers Only**: If you're completely new to open source, check out the [`first timers only`](https://github.com/chaoss/augur/labels/first%20timers%20only) label
+- **Work on Small Issues**: If you're feeling ready to give Augur development a try, pick a small issue to solve
+- **File New Issues**: If you think you found a problem that doesn't have an issue, file one and talk to us about it so we can confirm the issue and discuss how it should be fixed
+
+#### How to Find Good First Issues
+You can find good first issues by clicking on the "Issues" tab in the Augur repository and filtering by the `good first issue` label:
+
+<img width="1890" alt="GitHub issues page showing all open issues" src="https://github.com/user-attachments/assets/55b21b8a-df91-4c11-bbcd-38597f9d3cd8">
+
+Then filter by label to see only the `good first issue/ first-timer-only` items:
+
+<img width="1738" alt="GitHub issues filtered to show only good first issues" src="https://github.com/user-attachments/assets/f505dc2f-c034-4229-97da-98bc8173f713">
+
+<img width="1872" alt="GitHub issues filtered showing good first issue results" src="https://github.com/user-attachments/assets/c67cf2e8-0425-4451-9296-839bc9bf11f5">
```

Comment:
if we are going to include images, they should be uploaded to the repo so they dont disappear in future